src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
Windows is an outlier in this example since it maintains compat across releases. Even if we were to support Windows by keeping win7-x64 as the base RID, what would be our story for OSX and Linux as compat is not guaranteed there? And even on Windows, apps/libraries that have only, say, assets for win10-x64 but not for win7-x64 (since they added native dependency at a later point) will also not work if we set the base to win7-x64.

For unsupported platforms, we can say that DOTNET_RUNTIME_ID should be used to set the RID appropriate for each platform. IMHO, there is no clean/consistent story for them.